src/components/VideoList/index.jsx

```diff
@@ -1,22 +1,57 @@
-import { useQuery } from "@tanstack/react-query";
+import { useState } from "react";
 import { Link } from "react-router-dom";
+import ReactPlayer from "react-player/youtube";
 
 import useFetchSingleVideo from "../../apis/useFetchSingleVideo";
 
 function VideoList({ youtubeVideoId }) {
+  const [isHover, setIsHover] = useState(false);
+  const [isAvailable, setIsAvailable] = useState(true);
   const { data: video, isFetching } = useFetchSingleVideo(youtubeVideoId);
+  const youtubeURL = `https://www.youtube.com/watch?v=${youtubeVideoId}`;
 
   return (
     <div>
       {!isFetching && (
         <Link to={`/watch?${video.youtubeVideoId}`} state={{ video }}>
           <div className="flex justify-between w-screen gap-x-6 p-2">
             <div className="flex min-w-0 gap-x-4">
-              <img
-                className="h-[200px] flex-none rounded-md bg-gray-50"
-                src={video.thumbnailURL}
-                alt="thumbnail"
-              />
+              {isAvailable ? (
+                <div
+                  onMouseEnter={(event) => {
+                    setTimeout(() => {
+                      setIsHover(true);
+                    }, 300);
+                    event.stopPropagation();
+                  }}
```

Comment:
```suggestion
                  function handleMouseEnter(event) => {
                    setTimeout(() => {
                      setIsHover(true);
                    }, 300);
                    event.stopPropagation();
                  }
                  
                  onMouseEnter={handleMouseEnter}
```
이벤트 리스너의 콜백함수는 변수로 만드면 가독성이 더 좋을 것 같습니다!

Reply:
넵 수정하겠습니다!